pkg/managed/testutils.go

```diff
@@ -84,6 +85,63 @@ func (s simplePodNamespaceLister) Get(name string) (*v1.Pod, error) {
 	return s.kubeClient.CoreV1().Pods(s.namespace).Get(context.TODO(), name, metav1.GetOptions{})
 }
 
+// ----- simpleSecretLister
+// Simple secret sister implementation.
+type simpleSecretLister struct {
+	kubeClient kubernetes.Interface
+}
+
+// List all secrets
+func (s *simpleSecretLister) List(selector labels.Selector) (ret []*v1.Secret, err error) {
+	namespaces, err := s.kubeClient.CoreV1().Namespaces().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return nil, err
```

Comment:
The method signature has return names so I think you just do a return with no params.  See https://tour.golang.org/basics/7

Reply:
I see what you mean but from my perspective this makes the code harder to read.  I don't think I'll ever take advantage of this go "feature". ;)

Reply:
Then you can get rid of the named return params.